cluster-operator/src/main/java/io/strimzi/operator/cluster/model/TopicOperator.java

```diff
@@ -37,9 +37,6 @@
     // Configuration defaults
 
 
-    // Configuration keys
-    public static final String KEY_CONFIG = "topic-operator-config";
-
     // Topic Operator configuration keys
     public static final String KEY_CONFIGMAP_LABELS = "STRIMZI_CONFIGMAP_LABELS";
```

Comment:
Are these used for env vars? If so we should rename them to `ENV_VAR_...`

Reply:
Right. Done.